eultheme/templates/downtime/downtime.html

```diff
@@ -0,0 +1,269 @@
+
```

Comment:
Would it be very hard to add block tags to make this extendable if projects wanted to customize?